tests/copyoffload/test_copyoffload_migration.py

```
@@ -5362,3 +5364,275 @@ def test_check_vms_vddk(
+            target_namespace=target_namespace,
+            expected_xcopy_used=True,
+            fixture_store=fixture_store,
+        )
+
+    def test_check_vms(
+        self,
+        prepared_plan: dict[str, Any],
+        source_provider: BaseProvider,
+        destination_provider: OCPProvider,
+        source_provider_data: dict[str, Any],
+        target_namespace: str,
```

Ruff (0.15.21)

[warning] Unused method argument: `target_namespace`

(ARG002)